docs/docs/policies.mdx

```diff
@@ -157,13 +164,23 @@ If you want to fine-tune your model, start by modifying the following parameters
   Also, it is usual practice to have decreasing values in the list: next value is smaller or equal to the
   value before.
 
-* `number_of_transformer_layers`:
-  This parameter sets the number of transformer layers to use (default: `1`).
-  The number of transformer layers corresponds to the transformer blocks to use for the model.
+* `sequence_number_of_transformer_layers`:
```

Comment:
I am confused. This does not match the with the changes you mentioned earlier or am I missing something?

> number_of_transformer_layers -> dictionary “number_of_transformer_layers” with keys “text”, “action_text”, “label_action_text”, “dialogue”

Reply:
same for the others

Reply:
Ok, I think you simply forgot to update these here.

Reply:
yep simply forgot to remove